ddtrace/debugging/_encoding.py

```diff
@@ -110,9 +111,11 @@ def _build_log_track_payload(
     }
 
     # Add the correlation IDs if available
-    if context is not None:
-        payload["dd.trace_id"] = str(context.trace_id)
-        payload["dd.span_id"] = str(context.span_id)
+    if context is not None and context.trace_id is not None:
+        payload["dd"] = {
+            "trace_id": format_trace_id(context.trace_id),
```

Comment:
This looks to be a breaking change and this will now be the full 128-bit value, but I suspect this change is intended?

Reply:
Yes, this fixes the correlation between traces and dynamic instrumentation logs